src/main/java/nc/multiblock/turbine/Turbine.java

```diff
@@ -469,11 +469,15 @@ protected boolean isMachineWhole(IMultiblockValidator validatorCallback) {
 				return false;
 			}
 
-			totalExpansionLevel *= currentBlade.bladeType.getExpansionCoefficient();
-			expansionLevels.add(totalExpansionLevel);
+
+
 			if (currentBlade.isStator) {
+				totalExpansionLevel *= NCConfig.turbine_stator_expansion;
```

Comment:
Stators and their magic numbers, wish they could be a blade just like any other...